src/js/hexbin/HexbinLayer.js

```diff
@@ -302,10 +302,30 @@ L.HexbinLayer = (L.Layer ? L.Layer : L.Class).extend({
 		this.options.onclick = clickFn;
 		//this._redraw();
 		return this;
+	},
+
+	/*
+	 * Returns an array of the points in the path, or nested arrays of points in case of multi-polyline.
+	 */
+	getLatLngs: function () {
+		return this._data.map(function(d) { return L.latLng(d[0], d[1]); });
+	},
+
+	/*
+	 * Get path geometry as GeoJSON
+	 */
+	toGeoJSON: function () {
+		var latLngs = this._data.map(function(d) { return L.latLng(d[0], d[1]); });
```

Comment:
Yeah you're right, on closer inspection that whole function is a write off really... (it does not handle multi at all!) sorry for wasting your time mate. 

Unfortunately I don't have much free time to work on this as I'm on other projects now.. maybe leave the PR open though and I'll see if I can get back to it soon.

I'll see if I can get a screenshot example of the vis for you now.